test/unit/calculators/marriage_abroad_calculator_test.rb

```diff
@@ -938,6 +938,22 @@ class MarriageAbroadCalculatorTest < ActiveSupport::TestCase
           refute @calculator.ceremony_country_offers_pacs?
         end
       end
+
+      context '#french_overseas_territory_offering_pacs?' do
+        setup do
+          @calculator = MarriageAbroadCalculator.new
+        end
+
+        should 'return true if a PACS is available in the ceremony country and the country is a French overseas territory' do
+          @calculator.ceremony_country = 'new-caledonia'
+          assert @calculator.french_overseas_territory_offering_pacs?
+        end
+
+        should "return false if PACS is available in the ceremony country but it's not a French overseas territory" do
+          @calculator.ceremony_country = 'france'
+          refute @calculator.french_overseas_territory_offering_pacs?
+        end
+      end
```

Comment:
I don't understand the naming of these tests. The implementation only has a single condition, but the two test names mention two conditions. The test names might indeed by correct, but it would be good to explain how you derived them in the commit note.